includes/gateways/paypal/admin/settings.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * PayPal Settings
+ *
+ * @package   easy-digital-downloads
+ * @copyright Copyright (c) 2021, Sandhills Development, LLC
+ * @license   GPL2+
+ * @since     2.11
+ */
+
+namespace EDD\Gateways\PayPal\Admin;
+
+/**
+ * Register the PayPal Standard gateway subsection
+ *
+ * @param array $gateway_sections Current Gateway Tab subsections
+ *
+ * @since 2.11
+ * @return array                    Gateway subsections with PayPal Standard
+ */
+function register_paypal_gateway_section( $gateway_sections ) {
+	$gateway_sections['paypal_commerce'] = __( 'PayPal', 'easy-digital-downloads' );
+
+	return $gateway_sections;
+}
+
+add_filter( 'edd_settings_sections_gateways', __NAMESPACE__ . '\register_paypal_gateway_section', 1, 1 );
+
+/**
+ * Registers the PayPal Standard settings for the PayPal Standard subsection
+ *
+ * @param array $gateway_settings Gateway tab settings
+ *
+ * @since 2.11
+ * @return array Gateway tab settings with the PayPal Standard settings
+ */
+function register_gateway_settings( $gateway_settings ) {
+
+	$paypal_settings = array(
+		'paypal_settings'              => array(
+			'id'   => 'paypal_settings',
+			'name' => '<strong>' . __( 'PayPal Settings', 'easy-digital-downloads' ) . '</strong>',
```

Comment:
Should header "settings" use `h3` instead of `strong`? I've seen us use both and they're styled identically, but `h3` has semantic meaning.